docs/resources/cce_cluster.md

```diff
@@ -145,6 +145,12 @@ versions are available, choose Dashboard > Buy Cluster on the CCE console. Chang
 
 * `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id of the cce cluster. Changing this creates a new cluster.
 
+* `masters` - (Optional, List, ForceNew) Advanced configuration of master nodes. Changing this creates a new cluster. Conflicts with `multi_az`.
```

Comment:
seems `masters` and `multi_az` have the same logical function, so please put them together.

Reply:
fixed